packages/febusy/febusy.0.0.0/opam

```diff
@@ -13,9 +13,9 @@ build: [
   ["jbuilder" "build" "-p" "febusy" "-j" jobs ]
 ]
 depends: [
-  "ocaml" { >= "4.03.0" }
+  "ocaml" {>= "4.03.0" & < "5.0.0"}
```

Comment:
Is this needed? The error does not seem to be related to ocaml 5

Reply:
Ah indeed. That'll teach me that putting unrelated changes is risky.